Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -203,6 +205,10 @@ public void CalculateBoundaryVolume()
             // Ensuring that we set height of the bounds volume to be say 10 feet tall.
             boundaryBounds.Encapsulate(new Vector3(0, boundaryHeight, 0));
         }
+
+        protected override void InitializeInternal()
```

Comment:
This probably shouldn't be inside the block for `UNITY_2017_2_OR_NEWER`

Reply:
Thats moved in this PR: https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1518